### PR TITLE
support cocoapods static framework

### PR DESCRIPTION
**Pull request by looseyi**

### CODE REVIEW

File: supports/debug.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -114,8 +112,7 @@
 		38A90F89163506F8001DB09E /* debugTests */ = {
 			isa = PBXGroup;
 			children = (
-				38F85958215BB56300CD80AE /* debugTests.m */,
-				38A90F90163506F8001DB09E /* debugCppTests.mm */,
+				38A90F90163506F8001DB09E /* debugTests.mm */,
```
Comment (youknowone) [inline supports/debug.xcodeproj/project.pbxproj]: there were both `.mm` and `.m` tests to test objetive-c and objective-c++ actually.
Is there any reason to remove `.m` test?

**Review by youknowone — CHANGES_REQUESTED** [review]
Thanks for your contributing! I left a question about test file. will you check it please?

File: supports/debug.xcodeproj/project.pbxproj (hunk 1 above)
Comment (looseyi) [reply supports/debug.xcodeproj/project.pbxproj]: sorry, this change was make by the FoundationExtensions repo's cdebug. I'll reset this file's changed.

**Comment by youknowone** [comment]
Thanks for your contributing!

**Review by youknowone — APPROVED** [review]

**Pull request merged**